trafilatura/utils.py

```diff
@@ -454,3 +454,21 @@ def text_chars_test(string: Optional[str]) -> bool:
     # or not re.search(r'\w', string)
     # return string is not None and len(string) != 0 and not string.isspace()
     return bool(string) and not string.isspace()  # type: ignore[union-attr]
+
+
+def copy_attributes(dest_elem: _Element, src_elem: _Element) -> None:
+    '''Copy attributes from src element to dest element'''
+    for key in src_elem.keys():
```

Comment:
I would rather recommend something like this, I think it should work:
`dest_elem.attrib = src_elem.attrib`

Reply:
I tried at first, but attrib of _Element is not writable